actor/echo-messaging/wasmcloud.toml

```diff
@@ -0,0 +1,7 @@
+name = "{{project-name}}"
+language = "rust"
+type = "actor"
+version = "0.1.0"
+
+[actor]
```

Comment:
Regarding the first comment, just add this under the [actor] block for WASI compilation 
```
wasm_target = "wasm32-wasi"

```